### PR TITLE
Add github actions for rego files

### PR DESCRIPTION
**Pull request by DiamondJoseph**
- Test and lint rego files in org-policy directory on pushing to github
- Build and publish OCI Bundle when tag created on github
- Add OPA plugin to devcontainer

### CODE REVIEW

**Comment by DiamondJoseph** [comment]
I'm uncertain if the Policy release will occur when making a tag

File: .github/workflows/policy-release.yml
Comment (garryod) [inline .github/workflows/policy-release.yml:1]: Can we rename this file to be consistent with the bundler one?

File: .github/workflows/policy-release.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Policy Container
+
+on:
+  push:
+    branches:  # Release only when main advances
+      - main
```
Comment (garryod) [inline .github/workflows/policy-release.yml:6]: We want to run the CI on every push, just not do the upload

File: .github/workflows/policy-release.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      - name: Setup OPA
+        uses: open-policy-agent/setup-opa@v2
+        with:
+          version: latest
+      
+      - name: Build & Publish OPA Policy
+        run: opa build ./org-policy
+
+      - name: Publish OPA Policy
+        run: oras push ${{ env.IMAGE_REPOSITORY }}:${{ env.IMAGE_TAG }} --config ./org-policy/config.json:application/vnd.oci.image.config.v1+json bundle.tar.gz:application/vnd.oci.image.layer.v1.tar+gzip
```
Comment (garryod) [inline .github/workflows/policy-release.yml:40]: Should add an `if` here that is similar to the one on in `bundler-container`, e.g.:
```
${{ !(github.event_name == 'push' && startsWith(github.ref, 'refs/tags')) }}
```

File: .github/workflows/policy-release.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+      - name: Log in to GitHub Docker Registry
+        uses: docker/login-action@v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Setup OPA
+        uses: open-policy-agent/setup-opa@v2
```
Comment (garryod) [inline .github/workflows/policy-release.yml:32]: Would rather use a fully qualified version number

File: .github/workflows/policy-test.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  test:
+    # Deduplicate jobs from pull requests and branch pushes within the same repo.
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4.1.1
+
+      - name: Setup OPA
+        uses: open-policy-agent/setup-opa@v2
```
Comment (garryod) [inline .github/workflows/policy-test.yml:33]: Would rather use a fully qualified version number

**Review by garryod — CHANGES_REQUESTED** [review]

**Review by garryod — APPROVED** [review]
LGTM, any chance you could squash the commits down some before merging?

**Review by garryod — APPROVED** [review]

**Pull request merged**